Blockchain/readme.md

```diff
@@ -81,6 +83,7 @@
 -  [Ceramic Network](https://developers.ceramic.network/learn/welcome/)
 -  [The Graph](https://thegraph.com/docs/en/)
 -  [Scaffold-Ethereum](https://docs.scaffoldeth.io/scaffold-eth/)
+-  [Alchemy Dapp Store](https://www.alchemy.com/dapps)
```

Comment:
Not sure how this is a tool. Could you please explain how a store can come under the tools category?

Reply:
Actually its a store of dapps and tools . There are 1263 dapps and tools listed there . Its a one stop for web3 folks . They can explore according to their needs. New Dapps and Tools with 
Blockchains
DAO Tools
DAOs
DeFi Dapps
DeFi Tools
Decentralized Games
Identity Tools
Infrastructure Tools
NFT Dapps
NFT Tools
Solidity Tools
Trading Tools
Venture Capital Firms
Web3 Developer Tools
Web3 Gaming Tools
Web3 Security Tools
Web3 Social Dapps
Web3 Wallet Tools
Web3 Wallets
I hope you got the point of adding it in the list